GPSUARTDriver.device.lib.nut

```diff
@@ -0,0 +1,110 @@
+// MIT License
+// Copyright 2017-18 Electric Imp
+// SPDX-License-Identifier: MIT
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO
+// EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+// OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+// Driver for UART GPS modules. This driver is dependant on the GPSParser class.
+// This driver is focused on retrieving location data and returns data from the
+// GPSParser.getGPSDataTable method. Currently only the data with sentence IDs
+// VTG, RMC, GLL, GGA, GSV and GSA are supported.
+class GPSUARTDriver {
+
+    function _statics_() {
+        const VERSION          = "1.0.0";
```

Comment:
This way we make the `VERSION` unavailable from outside... So you can't write `GPSUARTDriver.VERSION` anymore (have anybody ever done it before?). Which is probably fine...

Reply:
... especially as there is `GPSParser.VERSION` available

Reply:
I can move this to a static variable